LogosLinuxInstaller.py

```diff
@@ -80,6 +83,14 @@ def parse_args(args):
     if args.reinstall_dependencies:
         config.REINSTALL_DEPENDENCIES = True
 
+    if args.get_winetricks: 
+       setWinetricks
+       sys.exit(0)
+
+    if args.run_winetricks:
+        run_winetricks()
+        sys.exit(0)
+
```

Comment:
If these functions are run here in `parse_args()`, then the config variables are not yet properly set when these functions run. This ties into discussion #34. At the point where `parse_args()` is called, the config file has been read, but other CLI options haven't yet been handled (e.g. `-V`) and the environment variables have not yet been read. This happens in this order so that environment variables take precedence over CLI options, which take precedence over the config file, which takes precedence over default config. So the CLI arguments have to be read and parsed, then the environment variables need to be read and parsed, then the runtime config can finally bet set, and only then can any "actions" run under the intended runtime config.

Reply:
See discussion